app/actions/local/group.ts

```diff
@@ -2,10 +2,12 @@
 // See LICENSE.txt for license information.
 
 import {fetchFilteredChannelGroups, fetchFilteredTeamGroups, fetchGroupsForAutocomplete} from '@actions/remote/groups';
+import {getOperator} from '@app/helpers/database';
```

Comment:
please update the branch and use the Database manager's new functions to get the operator/database instead of this.